crates/node-resource-metrics/src/collectors/basic_node_info_collector.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use once_cell::sync::Lazy;
+use prometheus::{
+    core::{Collector, Desc},
+    proto::MetricFamily,
+    register_int_gauge_vec, IntGaugeVec,
+};
+
+/// Current host name
+pub static HOST_NAME: Lazy<IntGaugeVec> = Lazy::new(|| {
+    register_int_gauge_vec!("aptos_node_host_name", "A no-op counter value", &[
```

Comment:
Two problems here:
1. We are double registering. This macro already registers the counter, and its being registered below in `register_basic_node_info_collectors`. We should not register here, but use a macro that doesn't register.
2. I don't see where `set(...)` is actually called on `HOST_NAME`. We can actually do this inside `collect()`.

Reply:
>We are double registering. This macro already registers the counter, and its being registered below in register_basic_node_info_collectors. We should not register here, but use a macro that doesn't register.

Good point - changed it. 

> I don't see where set(...) is actually called on HOST_NAME. We can actually do this inside collect()

Hah, my bad. I had changes locally which did that but missed to add it to the PR. Updated now.